nghttp/test_apps/main/nghttp_test.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
```

Comment:
sh2lib component is actually a wrapper on nghttp APIs and the example there should cover the tests part as well

Reply:
We have similar situation with dhara and spi_nand_flash. I thought about it for a bit and decided to add the app anyway for the following reason:
- there is a ever need to add a test for the underlying component rather than for the wrapper, the test app is already there and it is trivial to add the test
- if both apps get built, ccache will take care of reusing most object files, so building both apps shouldn't take too long

However I can remove the nghttp one if you think it's really not going to be needed.

Reply:
After some more thought I removed these two apps. We can easily re-add them later if needed...